src/mmw/mmw/settings/layer_settings.py

```diff
@@ -141,6 +141,7 @@
         'stream': True,
         'overlay': True,
         'minZoom': 5,
+        'maxZoom': 22,
```

Comment:
There's some code we have specifically for the dynamic nature of Google basemap max zoom:
https://github.com/WikiWatershed/model-my-watershed/blob/1a836bb5f8281b7d7f2b0bedb2a2f7bf9131794a/src/mmw/js/src/core/views.js#L359-L373